orttraining/orttraining/test/gradient/gradient_ops_test.cc

```diff
@@ -1761,6 +1761,18 @@ void record_event(int64_t event_id) {
   test_record.Run();
 }
 
+void record_event_multiple_inputs_and_outputs(int64_t event_id) {
```

Comment:
Please all the tests for record/wait into a separated test file.
This file is reserve for gradientop numerical tests

Reply:
Ok.